src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -300,24 +300,6 @@ public void testGetFirstNode2() {
             .isEqualTo(root);
     }
 
-    @Test
-    public void testIsReceiverParameter() throws Exception {
```

Comment:
We don't require this pure ut to cover coverage and mutation or even to test functionality